app/pages/multisig/updates/FinalizationCommitteeParameters/UpdateFinalizationCommitteeParameters.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { isChainParametersV0, isChainParametersV1 } from '@concordium/web-sdk';
+import { EqualRecord } from '~/utils/types';
+import { UpdateProps } from '~/utils/transactionTypes';
+import Form from '~/components/Form/';
+import { mustBeAnInteger, requiredMessage, enterHere } from '../common/util';
+import {
+    FinalizationCommitteeParametersFields,
+    fieldDisplays,
+    getCurrentFinalizationCommitteeParameters,
+} from './util';
+import ShowFinalizationCommitteeParameters from './FinalizationCommitteeParametersShow';
+import { FormRewardFractionField as FractionFieldForm } from '../common/RewardFractionField/RewardFractionField';
+
+const fieldNames: EqualRecord<FinalizationCommitteeParametersFields> = {
+    minFinalizers: 'minFinalizers',
+    maxFinalizers: 'maxFinalizers',
+    relativeStakeThresholdFraction: 'relativeStakeThresholdFraction',
+};
+
+const UINT32_MAX = 2 ** 32 - 1; // UInt32 upper bound
```

Comment:
This value has already been defined elsewhere as well. Refactor it to only be defined once.